parity/url.rs

```diff
@@ -16,33 +16,70 @@
 
 //! Cross-platform open url in default browser
 
+use std;
+
+#[allow(unused)]
+pub enum Error {
+	ProcessError(std::io::Error),
+	FfiNull(std::ffi::NulError),
+	WindowsShellExecute,
```

Comment:
I'd suggest putting the return value of `ShellExecuteA` (casted as a `c_int`) as parameter, for easier debugging if that actually happens.